.github/workflows/dev.yaml

```diff
@@ -5,53 +5,31 @@ on:
       - dev-cere
   workflow_dispatch:
 
+permissions:
+  id-token: write
+  contents: read
+  
 jobs:
   build_backend_cerestats_api:
-    uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-push-docker.yaml@1.0.0
+    uses: Cerebellum-Network/reusable-workflows/.github/workflows/deploy-to-ecr.yaml@1.3.0
     with:
       runs-on: '["self-hosted", "cere-network-large-workers"]'
-      org: cerebellumnetwork
       environment: dev
-      image: crb-cerestats-api
-      repository: crb-cerestats-api
+      aws_account_id: ${{ vars.DEV_NETWORK_AWS_ACCOUNT_ID }}
+      repository: stats-api
       file: ./api/Dockerfile
     secrets: inherit
-
+    
   build_backend_cerestats_crawler:
-    uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-push-docker.yaml@1.0.0
+    uses: Cerebellum-Network/reusable-workflows/.github/workflows/deploy-to-ecr.yaml@1.3.0
     with:
       runs-on: '["self-hosted", "cere-network-large-workers"]'
-      org: cerebellumnetwork
       environment: dev
-      image: crb-cerestats-crawler
-      repository: crb-cerestats-crawler
+      aws_account_id: ${{ vars.DEV_NETWORK_AWS_ACCOUNT_ID }}
+      repository: stats-crawler
       file: ./backend/docker/crawler/Dockerfile
     secrets: inherit
 
-  deploy_backend_cerestats_api:
```

Comment:
If we don't need the `deploy*` scripts, it may make sense to rename the jobs from `build_*` to `build_and_deploy_*`